### ISSUE TITLE
Nimrod ORMs

### ISSUE THREAD

**Issue by blokhin** [nanoant/nim-orm#1]
Good day!
First of all, thanks for your interesting repo. I've got just a small general question: are you aware of any other nim ORMs? This language seems to be very young (rather promising though!) and the tooling isn't developed enough yet.
Thanks again for your time.

**Comment by nanoant** [comment]
Nope, I am not aware of any other ORM. I invite you to #Nim channel at FreeNode, probably some other folks may hear or work on something similar.

Anyways my ORM tries to leverage the language features such as macros, AST rewriting, while you can much easier create a traditional ORM implementation, where most stuff is done at runtime level. But such ORM wouldn't be different from any other existing one :)
